### ISSUE TITLE
Loading animation is missing after redirected from the Native 3DS2 component

### ISSUE THREAD

**Issue by ahmetgsu** [Adyen/adyen-react-native#70]
**Describe the bug**

We recently upgraded the library from BETA.3 to BETA.7. After completing the Native 3DS2 flow in the app, the user is redirected to the DropIn component first, and then a few seconds later the DropIn disappears. In BETA.3 version, we had a loading animation before the DropIn disappears, However, this loading animation is missing in BETA.7 on IOS. On Android, we can see the loading spinner.

**To Reproduce**

1. Launch DropIn
2. Proceed to Native 3DS2 component, and Enter the 1234 code to finish this process,
3. See whether a loading spinner exists on Ios after the Native 3DS2 component is dismissed and the DropIn component appeared

**Expected behavior**
After completed the Native 3DS2 flow, we need to see a loading spinner until the DropIn is dismissed.

**Smartphone (please complete the following information):**
 - Device: IPhone 14 Pro emulator 
 - OS: IOS 16.1

**Comment by descorp** [comment]
Hey @ahmetgsu 

Thank you for staying with us for 4 beta versions :)
Could you provide a video/screenshot comparison?
I can't think of anything on the RN side that could have this effect...

I believe this causing UX issues on your side?
Surprisingly, the solution could be the same as for your previous suggested [enchantment](https://github.com/Adyen/adyen-react-native/issues/7) - we are looking to add a "stopLoading" feature that merchants can use to "unblock" UI, hence we are about to block UI while the payment is in progress

**Comment by ahmetgsu** [comment]
Hey @descorp 

Thank you for your response.

I attach both flows on android and on ios to compare. 

SS1: we enter card details,
SS2: Native 3DS2 component appears,
SS3: We enter code,
SS4: Native 3DS2 component is dismissed, and we are redirected to the DropIn component.

As you might see on the 4th screenshot, while we have a loading animation on android, we don't have it on ios. We had this loading animation with BETA.3

In this case, the user can interact with the app on ios on the 4th step. 

**Android flow**

<img src="https://user-images.githubusercontent.com/48296634/203350924-7ff4665c-65be-44cb-82db-4147f72761e5.png" width="200" /> <img src="https://user-images.githubusercontent.com/48296634/203350971-08904502-0bc0-451a-a9c5-8a339f5160f3.png" width="200" /> 
<img src="https://user-images.githubusercontent.com/48296634/203351002-f8d2e950-cff7-4729-a60f-9cdf84645745.png" width="200" /> <img src="https://user-images.githubusercontent.com/48296634/203351055-f435495b-dcab-4711-8972-87e741d17d6c.png" width="200" />


**IOS flow**

<img src="https://user-images.githubusercontent.com/48296634/203351324-bd22897c-d1eb-4edc-9453-aca3aace36dc.png" width="200" /> <img src="https://user-images.githubusercontent.com/48296634/203351328-0bcee20a-3242-4317-8858-aa127f29950e.png" width="200" />
<img src="https://user-images.githubusercontent.com/48296634/203351332-3b39c339-82c5-4551-9541-61003bbc811b.png" width="200" /> <img src="https://user-images.githubusercontent.com/48296634/203351336-aac32a01-b73c-471e-a07e-b1c52c816a88.png" width="200" />

**Comment by ahmetgsu** [comment]
Hey @descorp 
I do not think that we have the same issue here with my previous enchantment.

Here are the screenshots of the same flow with BETA.3

You can notice that we had a loading animation on the `Confirm preauthorization` button when we come back from the Native 3DS2 component with [BETA.3] version. However, this behavior is missing with the [BETA.7].

I hope this comparison made the issue clear.

<img src="https://user-images.githubusercontent.com/48296634/203977538-83e05635-216b-4993-bb1f-606c651bf4b3.png" width="200" /> <img src="https://user-images.githubusercontent.com/48296634/203977670-0cc8f223-14ab-439c-863e-154d14d082a3.png" width="200" /> 
<img src="https://user-images.githubusercontent.com/48296634/203977735-f566e96f-0f05-4907-8971-77ca3856cbc7.png" width="200" /> <img src="https://user-images.githubusercontent.com/48296634/203977778-42cfb6c5-5133-4424-aaf7-659aefce4950.png" width="200" />

**Comment by descorp** [comment]
Hey @ahmetgsu 

Currently we are aiming to give  control over the "loading" state on component: 
- to keep "loading" state of the button till payment is concluded.
- add option to stoping "loading" state

**Comment by descorp** [comment]
Hey @ahmetgsu 

~This seems fixed for Android on current beta-7.~
~Could you check with latest  release ?~
~Same for iOS - after `didProvide` is triggered UI is disabled.~

Fixed on iOS.
Will be release on beta-8.

**Comment by descorp** [comment]
Hey @ahmetgsu 

This turned to be a bug on [native iOS SDK](https://github.com/Adyen/adyen-ios/pull/1137).
Will be moved to **beta-9** to not postpone [beta-8](https://github.com/Adyen/adyen-react-native/milestone/3) release

**Comment by ahmetgsu** [comment]
Hey @descorp 
is there any update about the missing loading animation (that we had on BETA3) on IOS?

**Comment by descorp** [comment]
Hey @ahmetgsu 

not yet - iOS v4.11.0 was not released since.

Good news - iOS about to release `v5.2.0` and `4.11.0` is next in priority
We will keep you posted.

**Comment by descorp** [comment]
iOS [4.10.4](https://github.com/Adyen/adyen-ios/releases/tag/4.10.4) is released.
Will work on applying the fix

**Comment by ahmetgsu** [comment]
Hey @descorp 

I confirm that the loading animation on the Preauthorization button works now (with the latest release) after the user is redirected to DropIn from the 3DS2 screen.

**Comment by descorp** [comment]
Thanks for verifying @ahmetgsu 💚
